reana-ui/src/pages/signin/Signin.js

```diff
@@ -77,7 +77,13 @@ export default function Signin() {
           />
         )}
       </Segment>
-      {config.hideSignup && (
+      {config.hideSignup && config.cernSSO && (
```

Comment:
When having both local accounts and users authenticated through SSO (that is `cernSSO: true`, `localUsers: true` and `hideSignup: true`), the message "_Note that you need to hold an official CERN account in order to use reana.cern.ch service_" is shown. Should we tell the users they can request an account by contacting the administrators, like we do when `cernSSO: false`, `localUsers: true` and `hideSignup: true`?

Reply:
Thanks for spotting this! Working with 3 flags is like solving a three body problem. I've fixed it and included an image how it looks like in this case